pkg/test/e2eutil/wait_util.go

```diff
@@ -16,11 +16,14 @@ package e2eutil
 
```

Comment:
Tests file should finish with <name>_test.go

Reply:
See https://github.com/operator-framework/operator-sdk/pull/1143#discussion_r347997048